lib/checks/mobile/meta-viewport-scale.js

```diff
@@ -26,10 +26,12 @@ if (
 }
 
 if (!lowerBound && result['user-scalable'] === 'no') {
+	this.data(['user-scalable=no']);
```

Comment:
Why is this an array? Is so should we do a join in the doT templates?

Reply:
I'd followed another example, there can only be one result but that looked like what doT was expecting.